client/main/backstage.js

```diff
@@ -1372,10 +1383,16 @@ const BackstageModel = Backbone.Model.extend({
         this.requestImport(paths);
     },
     trySave: function(filePath, type) {
-        this.requestSave(filePath);
+        this.requestSave(filePath).catch((e) => {
+            this.set('activated', true);
+            this.set('operation', 'saveAs');
+        });
```

Comment:
you don't want to make these functions async, and use a try-catch?

Reply:
can this be merged now?